src/index.html

```diff
@@ -5,10 +5,59 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home" class="logo__link">
+        <img src="images/logo.png"
+          alt="logo"
+          class="logo__img"
+        >
```

Comment:
```suggestion
        <img 
          src="images/logo.png"
          alt="logo"
          class="logo__img"
        >
```
![image](https://github.com/mate-academy/layout_moyo-header/assets/96409563/fab3adb9-6f66-447a-ae4d-ac2793fc75d1)